### PR TITLE
feat(zoolander): modularized basement as global component

### PR DESCRIPTION
**Pull request by runrog**
1st of modularizing our global components.

You can see basement on the following zoolander page:
http://localhost:9000/derek/incubation/homepage

### CODE REVIEW

**Comment by WidgetsBurritos** [comment]
Can one of the admins verify this patch?

**Comment by WidgetsBurritos** [comment]
ok to test

**Comment by WidgetsBurritos** [comment]
add to whitelist

File: styleguide/_themes/derek/scss/_main.scss [hunk 1]
```diff
@@ -5,7 +5,7 @@
 @import 'globalElements/mainnav';
 @import 'globalElements/rug';
 @import 'globalElements/footer';
-@import 'globalElements/basement';
+@import '../../../derek/global-components/basement/basement';
```
Comment (amberRrucker) [inline styleguide/_themes/derek/scss/_main.scss]: in the future can we create a baseURL for this so the paths dont get so out of hand?

Comment (runrog) [inline styleguide/_themes/derek/scss/_main.scss]: If you mean, doing something like this
```
@import '#{$baseUrl}/component';
```
SASS won't let you use a variable inside an import

Comment (runrog) [inline styleguide/_themes/derek/scss/_main.scss]: would love to define that in place thought.

**Comment by WidgetsBurritos** [comment]
[run ci]

**Comment by WidgetsBurritos** [comment]
Since Travis is crapping out today you can't really see but you can see it on the CI job and also reproduce locally: 
```
davi0040@local:[...vi0040/github/zoolander]$ npm run harp:compile

> zoolander@0.17.0 harp:compile /Users/davi0040/github/zoolander
> npm run harp compile styleguide dist


> zoolander@0.17.0 harp /Users/davi0040/github/zoolander
> harp  "compile" "styleguide" "dist"

{
  "source": "Sass",
  "dest": "CSS",
  "filename": "/Users/davi0040/github/zoolander/styleguide/derek/global-components/rug/rug.scss",
  "lineno": 2,
  "name": "Sass Error",
  "message": "Undefined variable: \"$gray-dark\".",
  "stack": ".rug {\n  background-color: $gray-dark;\n  border-bottom: 1px solid $gray-base;\n}\n\n.rug-wrapper {\n  @include make-container;\n  padding: 0;\n}\n\n.rug-headerWrapper {\n  @include make-lg-column(1);\n  @include make-md-column(2.75);\n  padding: 15px 25px 20px;\n}\n\n.rug-headerWrapperBorder {\n  border-left: 1px solid $gray-base;\n  padding-left: 15px;\n}\n\n.rug-head {\n  color: $white;\n  margin: 0;\n  padding: 0;\n}\n\n.rug-subhead {\n  color: $white;\n  font-size: 1.05rem;\n  line-height: 1rem;\n  margin: 0;\n  padding: 0;\n}\n\n.rug-subheadLink {\n  color: $white;\n\n  &:hover {\n    color: $gray-light;\n  }\n}\n\n.rug-contactOption {\n  @include make-lg-column(2);\n  @include make-md-column(2);\n  margin-top: 5px;\n  padding: 15px 0;\n}\n\n.rug-iconWrapper {\n  @include make-lg-column(1);\n  @include make-md-column(1);\n}\n\n.rug-icon {\n  color: $white;\n}\n\n.rug-iconPhone {\n  @include rotate(20deg);\n  margin: 2px 0 0 15px;\n}\n\n.rug-link {\n  @include make-lg-column(9);\n  @include make-md-column(8);\n  @include subhead;\n  color: $white;\n  cursor: pointer;\n  font-size: 2rem;\n  line-height: 3.45rem;\n  text-decoration: none;\n}\n\n.rug-link-hover {\n  &:hover,\n  &:focus {\n    color: $gray-light;\n    text-decoration: none;\n  }\n}\n\n@media only screen and (max-width: $screen-md-max) {\n  .rug-subhead {\n    font-size: 1rem;\n  }\n}\n\n@media only screen and (max-width: $screen-sm-max) {\n  @include responsive-invisibility('.rug');\n}\n"
}
Segmentation fault: 11
```

**Comment by runrog** [comment]
@WidgetsBurritos checking

**Comment by WidgetsBurritos** [comment]
[run ci]

**Pull request merged**